src/templates/pages/home.pug

```diff
@@ -123,34 +123,30 @@
               //-   a.nav-link(href='https://www.facebook.com/webpixels', target='_blank')
               //-     i.fe.fe-facebook
           .col-lg-2.col-6.col-sm-4.ml-lg-auto.mb-5.mb-lg-0
-            h6.heading.mb-3 Use Cases
+            h6.heading.mb-3 Information For
             ul.list-unstyled
               li
                 a(href='#tabs-individuals', data-toggle='view') Individual
               li
-                a(href='#tabs-businesses', data-toggle='view') Business or Location
+                a(href='#tabs-businesses', data-toggle='view') Locations
               li
-                a(href='#tabs-healthcare', data-toggle='view') Healthcare Provider
+                a(href='#tabs-healthcare', data-toggle='view') Testing Facilities
               li
-                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") Public Official
+                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") Community
           .col-lg-2.col-6.col-sm-4.mb-5.mb-lg-0
-            h6.heading.mb-3 About
+            h6.heading.mb-3 Zerobase Foundation
             ul.list-unstyled
+
               li
-                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") How to Deploy
-              li
-                a(href='#tabs-privacy', data-toggle='view') Privacy Policy
-              li
-                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") HIPAA
-              li
-                a(href='#tabs-privacy', data-toggle='view') GDPR
+                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") About us
               li
-                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") Roadmap
-          .col-lg-2.col-6.col-sm-4.mb-5.mb-lg-0
-            h6.heading.mb-3 Foundation
+                a(href='https://github.com/zerobase-io/smart-tracing/wiki', target="_blank") Privacy Policy
```

Comment:
This should point to the site Privacy Policy tab/page. Please see line 142 on how to point to this tab